### PR TITLE
Fetch tor targets via authenticated orchestra

### PR DESCRIPTION
**Pull request by bassosimone**
Closes https://github.com/ooni/orchestra/issues/82

### CODE REVIEW

File: orchestrate/ooni-orchestrate.toml.example [hunk 1]
```diff
@@ -19,3 +19,6 @@ url = "postgres://user:pass@hostname:port/dbname?sslmode=disable"
 
 [psiphon]
 config-file = "CHANGEME"
+
+[tor]
+targets-file = "CHANGEME"
```
Comment (hellais) [inline orchestrate/ooni-orchestrate.toml.example]: Can you create an example file and commit it?

Comment (bassosimone) [reply orchestrate/ooni-orchestrate.toml.example]: Sure!

Comment (bassosimone) [reply orchestrate/ooni-orchestrate.toml.example]: * file location: https://github.com/ooni/orchestra/tree/master/orchestrate

* file name: `tor-config.json.example`

Comment (bassosimone) [reply orchestrate/ooni-orchestrate.toml.example]: Done!

**Comment by bassosimone** [comment]
@hellais please take another look!

File: orchestrate/tor-config.json.example [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "f372c264e9a470335d9ac79fe780847bda052aa3b6a9ee5ff497cb6501634f9f": {
+    "address": "38.229.1.78:80",
+    "params": {
```
Comment (hellais) [inline orchestrate/tor-config.json.example]: We should also consider, that from the perspective of somebody doing sanitisation, you may also need/want to have the unhashed fingerprint.

File: orchestrate/tor-config.json.example [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+      "cert": [
+        "Hmyfd2ev46gGY7NoVxA9ngrPF2zCZtzskRTzoWXbxNkzeVnGFPWmrTtILRyqCTjHR+s9dg"
+      ],
+      "iat-mode": [
+        "1"
+     ],
+  }
+}
```
Comment (hellais) [inline orchestrate/tor-config.json.example]: We should probably add the semantics of this file to the ooni/spec too.

File: orchestrate/tor-config.json.example (hunk 2 above)
Comment (bassosimone) [reply orchestrate/tor-config.json.example]: Yes, good point. So, I have modified the example in ooni/spec to include this information and I will update this specification before merging the PR.

**Comment by bassosimone** [comment]
Waiting for travis to complete and then merging!

**Pull request merged**